README.rst

```diff
@@ -80,3 +80,27 @@ The package is jointly developed by the `Center for Coastal and Ocean Mapping, U
     :alt: joint efforts
 
 For further information, visit the `manual <https://www.hydroffice.org/manuals/ssm2/index.html>`_.
+
+Development quick-start on Ubuntu 24.04
```

Comment:
Added a OS-agnostic section in the manual: https://www.hydroffice.org/manuals/ssm2/stable/developer_guide_how_to_contribute.html#development-quick-start